shop/src/utils/useCurrencyOpts.js

```diff
@@ -0,0 +1,34 @@
+import get from 'lodash/get'
+
+import { useStateValue } from 'data/state'
+
+import usePrice from './usePrice'
+import useConfig from './useConfig'
+
+const useCurrencyOpts = () => {
```

Comment:
Nit: document what this function does. Sorry for being a pita about my review comments... I just feel having some level of commenting in the source code makes it lot easier to maintain it in the future :)